operate/services/manage.py

```diff
@@ -232,6 +237,44 @@ def create(
 
         return service
 
+    def _compute_service_env_variables(
+        self, service: Service, staking_params: t.Dict[str, str]
+    ) -> None:
+        """Compute values to override service.yaml variables for the deployment."""
+
+        # TODO A customized, arbitrary computation mechanism should be devised.
+        computed_values = {
+            "ETHEREUM_LEDGER_RPC": PUBLIC_RPCS[ChainType.ETHEREUM],
```

Comment:
ideally we can remove this hardcode rpcs from middleware at some point
and instead use valory endpoints from .env, the current PUBLIC_RPCS are public/rate-limited
may be inconsistent with tenderly forks, if they're not overwritten too

Reply:
Agree, there is a refactor task to do in the RPCs. Surprisingly "PUBLIC_RPCs" here are overriden by the environment variables GNOSIS_DEV_RPC, etc.

This is task of another PR, though.